### PR TITLE
cksum: add tests and fixtures

### PR DESCRIPTION
**Pull request by ycd**
Closes: https://github.com/uutils/coreutils/issues/1853

I'd also love to add a test with a file that has a slightly bigger byte-count, thought about running this with `scene_cmd!` with the following command `printf '\\%03o' $(seq 0500)'`  but i'm not sure how will it react the CI environment.

### CODE REVIEW

File: tests/by-util/test_cksum.rs [hunk 1]
```diff
@@ -24,3 +24,73 @@ fn test_stdin() {
...
+
+#[test]
+fn test_invalid_file() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    let folder_name = "a".to_string();
+    at.mkdir(&folder_name);
+
+    ucmd.arg(&folder_name)
+        .fails()
```
Comment (sylvestre) [inline tests/by-util/test_cksum.rs:58]: please run rustfmt

**Comment by sylvestre** [comment]
MinRust is failing:
```
---- test_cksum::test_arg_overrides_stdin stdout ----
current_directory_resolved: 
touch: /tmp/.tmpuDYKuQ/a
run: /home/runner/work/coreutils/coreutils/target/debug/coreutils cksum a
thread 'test_cksum::test_arg_overrides_stdin' panicked at 'Broken pipe (os error 32)', tests/common/util.rs:574:37
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by sylvestre** [comment]
Fails on freebsd:
```
---- test_cksum::test_invalid_file stdout ----
current_directory_resolved: 
mkdir: /tmp/.tmpxqFke1/a
run: /tmp/cirrus-ci-build/target/debug/coreutils cksum a
thread 'test_cksum::test_invalid_file' panicked at 'assertion failed: !self.success', tests/common/util.rs:86:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

You should display stderr/stdout

**Comment by sylvestre** [comment]
On freebsd:
```


---- test_cksum::test_invalid_file stdout ----
current_directory_resolved: 
mkdir: /tmp/.tmpLie5OG/asdf
run: /tmp/cirrus-ci-build/target/debug/coreutils cksum asdf
"1160721993 512 asdf\n"
"1160721993 512 asdf\n"
thread 'test_cksum::test_invalid_file' panicked at 'assertion failed: result.stderr.contains(\"cksum: error: \'asdf\'\")', /tmp/cirrus-ci-build/tests/by-util/test_cksum.rs:61:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

**Comment by ycd** [comment]
Once #1951 is merged, i will push another empty commit to retrigger tests :)

**Review by Arcterus — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

**Comment by sylvestre** [comment]
```
---- test_cksum::test_arg_overrides_stdin stdout ----

current_directory_resolved: 

touch: /tmp/.tmpzZBPJm/a

run: /home/travis/build/uutils/coreutils/target/debug/coreutils cksum a

thread 'test_cksum::test_arg_overrides_stdin' panicked at 'Broken pipe (os error 32)', tests/common/util.rs:733:37

note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

failures:

    test_cksum::test_arg_overrides_stdin
```
fails

**Comment by ycd** [comment]
I have no idea why do i get broken pipe sometimes, am i doing something wrong?

**Comment by sylvestre** [comment]
> I have no idea why do i get broken pipe sometimes, am i doing something wrong?

Comment and disable the test for freebsd :)

**Comment by ycd** [comment]
@sylvestre should i retrigger the CI with an empty commit?, CI fails for `ubuntu-18.04, i686-unknown-linux-gnu` fails for

**Pull request merged**